apps/server/src/modules/files-storage/entity/filerecord.entity.ts

```diff
@@ -293,4 +293,10 @@ export class FileRecord extends BaseEntityWithTimestamps {
 
 		return PreviewStatus.PREVIEW_NOT_POSSIBLE_SCAN_STATUS_ERROR;
 	}
+
+	public get fileNameWithoutExtension(): string {
+		const fileNameWithoutExtension = this.name.split('.')[0];
```

Comment:
Das kann auch ein undefined zurück liefern, wenn nach dem Split in [0] nichts vorhanden ist.

Reply:
Nein? Wir haben immer ein string im name auch wenn das leer ist wird zu ""

Reply:
im spec ist auch ein test dazu

Reply:
Stimmt das liefert immer "" zurück.
Nur .bild.123.jpg als Test hinfügen, sollte dann .bild.123 zurück liefern.